tasks/wdl/ImputationBeagleTasks.wdl

```diff
@@ -356,7 +356,7 @@ task Impute {
     Int end
     Boolean impute_with_allele_probabilities
 
-    String beagle_docker = "us.gcr.io/broad-gotc-prod/imputation-beagle:2.0.0-d820c4e-1763720636"
+    String beagle_docker = "mbemis/mb-beagle:b00155d"
```

Comment:
placeholder

Reply:
at what point is it acceptable to build and push the new image to gcr? i also need to update a few pointers to the image in my warp-tools pr once that's done

warp-tools pr: https://github.com/broadinstitute/warp-tools/pull/177